app/src/main/java/org/oppia/app/options/StoryTextSizeActivityPresenter.kt

```diff
@@ -0,0 +1,87 @@
+package org.oppia.app.options
+
+import android.content.Intent
+import android.widget.SeekBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.app.databinding.StoryTextSizeActivityBinding
+import javax.inject.Inject
+
+/** The presenter for [StoryTextSizeActivity]. */
+@ActivityScope
+class StoryTextSizeActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity
+) {
+  private var fontSize: String = "Small"
+
+  fun handleOnCreate(prefSummaryValue: String) {
+    val binding =
+      DataBindingUtil.setContentView<StoryTextSizeActivityBinding>(activity, R.layout.story_text_size_activity)
+
+    fontSize = prefSummaryValue
+
+    binding.storyTextSizeToolbar.setNavigationOnClickListener {
+      val intent = Intent()
+      intent.putExtra("MESSAGE", fontSize)
+      (activity as StoryTextSizeActivity).setResult(1, intent)
+      activity.finish()
+    }
+
+    when (prefSummaryValue) {
+      "Small" -> {
```

Comment:
Can we define these values somewhere else in such a way that only constants are used and the hardcoded strings can be avoided in such critical places.

Reply:
Made use of constants.